src/lib/dnssd/Advertiser_ImplMinimalMdns.cpp

```diff
@@ -883,8 +904,10 @@ void AdvertiserMinMdns::AdvertiseRecords()
         packetInfo.DestPort  = kMdnsPort;
         packetInfo.Interface = interfaceAddress.GetInterfaceId();
 
+        // Advertise all records
+        // TODO: for kStarted,  don't announce records that haven't been updated.
```

Comment:
This should come with a comment about how we should also not announce them with a 0 TTL earlier or something?  Otherwise we will announce them with 0 TTL, then not announce here and then they are just gone, unless I am missing something.